### PR TITLE
Refine service account creation command

### PR DESCRIPTION
**Pull request by toanalien**
- Remove deprecated flag from service account creation script

### CODE REVIEW

File: collector/Dockerfile [hunk 1]
```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM otel/opentelemetry-collector-contrib:0.87.0
+FROM otel/opentelemetry-collector-contrib:0.100.0
```
Comment (damemi) [inline collector/Dockerfile:15]: is this relevant to the service account command change? If not, have you verified that all of the configs are still valid with this version of the collector?

We do need to update this but it should probably be a separate PR because it could break the provided configs, so those all need to be tested

**Review by damemi — CHANGES_REQUESTED** [review]

File: collector/Dockerfile (hunk 1 above)
Comment (toanalien) [reply collector/Dockerfile:15]: Hi @damemi,

You are correct! This should be separate another PR. :)

**Comment by damemi** [comment]
Replaced by https://github.com/GoogleCloudPlatform/opentelemetry-cloud-run/pull/23
